cfpb/consumerfinance.gov
# Resolve homebuying todo

Removes TODO sections since redirects have now been added in Wagtail

## Removals

- Removed homebuying temporary redirects in code since redirects were added in Wagtail


## How to test this PR

1. Ensure redirects for the following still work:
from: `/owning-a-home/prepare` to: `/owning-a-home/process/prepare`
from: `/owning-a-home/explore` to: `/owning-a-home/process/explore`
from: `/owning-a-home/compare` to: `/owning-a-home/process/compare`
from: `/owning-a-home/close` to: `/owning-a-home/process/close`
from: `/owning-a-home/sources` to: `/owning-a-home/process/sources`

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
